lib/src/extras/ws_rpc_server.rs

```diff
@@ -0,0 +1,20 @@
+use ws_rpc_server::WsRpcServer;
+
+use crate::error::Error;
+use crate::client::Client;
+use crate::config::config::WsRpcServerConfig;
+use crate::config::consts::default_bind;
+
+pub fn initialize_ws_rcp_server(client: &Client, config: WsRpcServerConfig) -> Result<WsRpcServer, Error> {
+    let ip = config.bind_to.unwrap_or_else(default_bind);
+
+    info!("Initializing websocket RPC server: {}:{}", ip, config.port);
+
+    let server = WsRpcServer::new(ip, config.port)?;
+    server.register_blockchain(client.consensus());
+    if let Some(validator) = client.validator() {
```

Comment:
This needs to be behind a feature flag.

Reply:
No, because the whole module is behind a feature flag.

Reply:
The validator part must be behind a feature gate. Fixed it.